exp/sampler.jl

```diff
@@ -140,6 +146,9 @@ if abspath(PROGRAM_FILE) == @__FILE__
         push!(D["input"]["meta"]["seed"], s)
         push!(D["input"]["data"]["pd"], data_.pd)
         push!(D["input"]["data"]["qd"], data_.qd)
+        push!(D["input"]["data"]["rmin"], data_.rmin)
+        push!(D["input"]["data"]["rmax"], data_.rmax)
+        push!(D["input"]["data"]["reserve_requirement"], data_.minimum_reserve)
```

Comment:
```suggestion
        push!(D["input"]["data"]["minimum_reserve"], data_.minimum_reserve)
```
All other keys match the field name of OPFData

Reply:
There is one thing I dislike about "minimum_reserve": it may be ambiguous whether it refers to the total amount of reserve that needs to be produced, or to the (per-generator) minimum reserve dispatch (which we currently denote by `rmin`).

I originally used the key `MRR`, but then went with `reserve_requirement` to avoid acronyms.

Reply:
updated

Reply:
LGTM